dashboard/test/ui/features/footer.feature

```diff
@@ -239,7 +239,7 @@ Feature: Checking the footer appearance
 
     # Additional wait to let scroll position settle and possibly have the
     # pin-to-home-screen popup go away
-    And I wait for 10 seconds
+    And I wait for 20 seconds
```

Comment:
Hmmm...this really seems unfortunate. 10s was already a pretty long wait. That said, I don't really have any better suggestions :(